fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -3,16 +3,20 @@
 
 import pandas as pd
 import numpy as np
+
 from .moment import ClassificationMoment
 from .moment import _GROUP_ID, _LABEL, _PREDICTION, _ALL, _EVENT, _SIGN
 from fairlearn._input_validation import _MESSAGE_RATIO_NOT_IN_RANGE
 from .error_rate import ErrorRate
 
+
 _UPPER_BOUND_DIFF = "upper_bound_diff"
 _LOWER_BOUND_DIFF = "lower_bound_diff"
 _MESSAGE_INVALID_BOUNDS = "Only one of difference_bound and ratio_bound can be used."
 _DEFAULT_DIFFERENCE_BOUND = 0.01
 
+_KW_CONTROL_FEATURES = "control_features"
```

Comment:
I know this is a duplicate... would need to think about injecting this in a sensible way